packages/cms-data-sync/scripts/run-calendar-sync-all.ts

```diff
@@ -0,0 +1,12 @@
+/* istanbul ignore file */
+import { getPrettyLogger } from '@asap-hub/server-common';
+import { getCalendarDataProvider } from '@asap-hub/crn-server/src/dependencies/calendars.dependencies';
+import { syncCalendars } from '../src/calendars/sync-all';
+
+const logLevel = process.env.LOG_LEVEL || 'info';
+const logEnabled = !!process.env.LOG_ENABLED || true;
+
+const logger = getPrettyLogger({ logEnabled, logLevel });
+const dataProvider = getCalendarDataProvider();
+
+syncCalendars({ dataProvider, logger }).catch(console.error);
```

Comment:
This should probably exit with a non-zero code if there's an error so that the build job fails.